uaclient/defaults.py

```diff
@@ -58,3 +58,5 @@
 
 ROOT_READABLE_MODE = 0o600
 WORLD_READABLE_MODE = 0o644
+NOTICES_PERMANENT_DIRECTORY = DEFAULT_DATA_DIR + "/notices/"
+NOTICES_TEMPORARY_DIRECTORY = "/var/run/notices/"
```

Comment:
Let's add `pro` prefix here to the folder, just to make it clearer to users that this is related to our tool